File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/SymbolOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class SymbolOutputDTO
    
        <[Parameter]("string", "", 1)>
        Public Property ReturnValue1 As String
    
    End Class

End Namespace

File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/SymbolOutputDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [FunctionOutput]
    public class SymbolOutputDTO
    {
        [Parameter("string", "", 1)]
        public string B {get; set;}
    }
}
